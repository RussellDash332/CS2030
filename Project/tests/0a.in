0.500
0.600
0.700
^D