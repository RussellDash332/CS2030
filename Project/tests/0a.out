0.500 1 arrives
0.600 2 arrives
0.700 3 arrives